plugin/evm/config.go

```diff
@@ -191,6 +191,11 @@ type Config struct {
 	// Database Settings
 	InspectDatabase bool `json:"inspect-database"` // Inspects the database on startup if enabled.
 
+	// OffChainWarpMessagesBytes represent off-chain warp messages (represented in hex) a validator is willing to sign.
+	// These messages are not tied to on-chain events and are particularly useful for implementing a unified
+	// upgrade mechanism for critical smart contracts on the chain.
```

Comment:
```suggestion
	// OffChainWarpMessagesBytes represent off-chain warp messages (represented in hex) a validator is willing to sign.
	// These messages are not tied to on-chain events.
```

I don't think we should specify the part about upgrading smart contracts on chain in the comment here. It is not enough detail to fully explain how they would be used, so I think it's better to omit the comment.

Reply:
Can we also switch to using `hexutil.Bytes` rather than a string so that we don't need to manually decode it?

Reply:
done